### ISSUE TITLE
AG WG Review: 2.4.12 Focus Not Obscured (Enhanced) (Level AAA) WCAG2ICT Guidance

### ISSUE THREAD

**Issue by pday1** [w3c/wcag2ict#927]
## Summary
Review of proposed WCAG2ICT guidance for the Level AAA WCAG SC: 2.4.12 Focus Not Obscured (Enhanced), as incorporated into the Editor's draft. 

## Maturity Level
Refining

## Action Needed
* Review content (See Content for Review section below for details.)
* Use the thumbs up and thumbs down icons to indicate agreement or disagreement with various comments. Do not add comments to simply support other comments. 
* Note any suggested edits for improvement and/or reasoning either in a comment on this issue or by creating a PR on the WCAG2ICT document.  
* Level A and AA SCs can be found in the [comments-by-guideline-and-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-by-guideline-and-success-criterion.md) file.
* Level AAA SCs can be found in the [comments-on-level-AAA-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-on-level-AAA-success-criteria.md)
----

## Background information 

Once finalized, this new WCAG2ICT Note will build on the previous version by also covering WCAG 2.2 level AAA success criteria (currently only level A and AA criteria are covered.  This is important as different jurisdictions are using different versions of WCAG for the accessibility of information and communication technology. 

## Content for Review

* Section [Applying 2.4.12 Focus Not Obscured (Enhanced) to non-web documents and non-web software](https://w3c.github.io/wcag2ict/#applying-2-4-12-focus-not-obscured-enhanced-to-non-web-documents-and-non-web-software).  

## Reference Material
In case you need them, links to various resources:

* [2.4.12 Focus Not Obscured (Enhanced)](https://www.w3.org/TR/WCAG22/#focus-not-obscured-enhanced) SC text in WCAG 2.2
* [Intent section of Understanding 2.4.12 Focus Not Obscured (Enhanced)](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-enhanced#intent)

**Comment by maryjom** [comment]
WCAG2ICT Note 1 seems like it was directly copied from SC 2.1.2 No Keyboard Trap Note 3 and isn't fully applicable to 2.4.12 Focus Not Obscured (Enhanced)
1. The 2nd sentence of this note is mainly about the management of keyboard focus - unrelated to this SC which is about the focus being fully visible (not obscured at all). 
2. The last sentence of this note is not correct for 2.4.12 Focus Not Obscured because it's talking about keyboard trap instead of this SC (emphasized text below highlights the problematic text):
>In this case, there is no concept of focus, and therefore **keyboard traps cannot exist**and this success criterion would be satisfied.

So that sentence should either be adjusted for this SC or removed. 

I also noticed that WCAG2ICT doesn't have any notes at all on SC 2.4.11 Focus Not Obscured (Minimum), so if this SC will have a note, it might also be applicable to 2.4.11.
